package.json

```diff
@@ -26,7 +26,8 @@
     "prebuild:js": "yarn test",
     "build:js": "node ./scripts/build.js",
     "pretest": "node ./scripts/pretest.js",
-    "test": "node ./scripts/test.js"
+    "test": "node ./scripts/test.js",
+    "lint": "./node_modules/.bin/tslint --project ."
```

Comment:
Thanks for adding this as a task. It would be great if this was also called in the `prebuild:js` task so that this is run each time a new build is compiled to help make sure Lint checks happen before each build.

Reply:
👍Added it to the prebuild script